core/src/test/java/org/mapfish/print/map/geotools/FeaturesParserTest.java

```diff
@@ -193,22 +193,14 @@ public void testTreatStringAsGeoJson() throws Exception {
       try {
         int numFeatures = getNumExpectedFeatures(geojsonExample);
         final String geojson =
-            new String(
-                java.nio.file.Files.readAllBytes(geojsonExample.toPath()),
-                Constants.DEFAULT_CHARSET);
-
+            java.nio.file.Files.readString(geojsonExample.toPath(), Constants.DEFAULT_CHARSET);
         final SimpleFeatureCollection simpleFeatureCollection =
             featuresParser.treatStringAsGeoJson(geojson);
         assertEquals(geojsonExample.getName(), numFeatures, simpleFeatureCollection.size());
-      } catch (AssertionError e) {
-        throw e;
-      } catch (Throwable t) {
-        t.printStackTrace();
-        throw new AssertionError(
-            "Exception raised when processing: "
-                + geojsonExample.getName()
-                + "\n"
-                + t.getMessage());
+      } catch (RuntimeException t) {
```

Comment:
Not sure but shouldn't be Exception?

Reply:
None of the code generate a checked exception, so it is not required. It will help in the future if someone introduce a checked exception to see that this code might need to be changed to handle differently the new exception.